mksusecd

```diff
@@ -1775,6 +1775,18 @@ sub run_createrepo
   print $_ if $opt_verbose >= 2 || !$ok;
 
   die "error: createrepo failed\n" if !$ok;
+
+  # sign repomd.xml
+
+  my $name = "$dir/repodata/repomd.xml";
+
+  return if !$sign_key_dir || !-f $name;
+
+  system "cp $sign_key_pub $name.key";
```

Comment:
It cannot have any strange character inside? I think that at least using `''` can help`

Reply:
No, it's a temp file name.

Reply:
e.g. if `$dir` contain space